internal/bull/notfound.go

```diff
@@ -40,7 +40,10 @@ bull works with pages, so maybe you would like to:
 func (b *bullServer) renderNotFound(w http.ResponseWriter, r *http.Request) error {
 	pageName := pageFromURL(r)
 	var buf bytes.Buffer
-	fmt.Fprintf(&buf, "# Error: page %q not found\n", pageName)
+	fmt.Fprintf(&buf, `# Error: page %q not found
+
+To create this page <a href="%s/edit/%s">click here</a> or press <kbd>Ctrl/Meta<kbd> + <kbd>E</kbd>.
+	`, pageName, b.URLBullPrefix(), pageName)
```

Comment:
The last parameter should better be `url.PathEscape(pageName)`. As with the other comment, browsers are pretty good at correcting non-escaped names, but it won’t necessarily always work.

Reply:
Thank you, fixed! :)